packages/valory/services/impact_evaluator/service.yaml

```diff
@@ -316,6 +322,8 @@ extra:
         max_points_per_period: ${MAX_POINTS_PER_PERIOD:int:5000}
         openai_call_window_size: ${OPENAI_CALL_WINDOW_SIZE:float:3600.0}
         openai_calls_allowed_in_window: ${OPENAI_CALLS_ALLOWED_IN_WINDOW:int:100}
+        twitter_call_window_size: ${TWITTER_CALL_WINDOW_SIZE:float:86400.0}
+        twitter_calls_allowed_in_window: ${TWITTER_CALLS_ALLOWED_IN_WINDOW:int:2}
```

Comment:
We only make 2 calls per day fetching 200 tweets a day for now, this means the amount of tweets pulled in the month would be `240000 (200 * 30 * 4)` so I would suggest we change this after a while because we only have quota of `10,000` tweets per month